src/services/support_documentation_service.py

```
@@ -0,0 +1,1070 @@
+        if filters.version:
+            filter_params["filter_version"] = filters.version
+
+        if filters.severity_level:
+            filter_params["filter_severity"] = filters.severity_level
+
+        if filters.document_category:
+            filter_params["filter_category"] = filters.document_category
+
+        return filter_params
+
+    async def _rerank_results(self, query: str, results: list[SearchResult]) -> list[SearchResult]:
```

Pylint (3.3.7)

[warning] Unused argument 'query'

(W0613)